tools/build/setup_macos_android.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+#
+# Copyright 2022 The Outline Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+# Android SDK Build Tools:
+#   https://developer.android.com/studio/releases/build-tools.html
+# To find the latest version's label:
+#   sdkmanager --list|grep build-tools
+ANDROID_BUILD_TOOLS_VERSION=${ANDROID_BUILD_TOOLS_VERSION:-"30.0.3"}
+
+# NDK (side by side) version must be kept in sync with the default build tools NDK version.
+NDK_VERSION=${NDK_VERSION:-"21.0.6113669"}
+
+function install_jdk() {
+  # Cordova Android 10 has to use JDK 11.
+  if [[ -d "$HOME/Library/Java/JavaVirtualMachines/jdk-11.0.2.jdk" ]]; then
```

Comment:
This might not work, my JDK is not installed here (it's installed by brew). We may use either `java --version` (looking for keyword `JDK` & `11`) or the compiler `javac --version` to determine whether JDK is installed:

```sh
java --version 
openjdk 11.0.16.1 2022-07-19 LTS
OpenJDK Runtime Environment Zulu11.58+23-CA (build 11.0.16.1+1-LTS)
OpenJDK 64-Bit Server VM Zulu11.58+23-CA (build 11.0.16.1+1-LTS, mixed mode)
```

```sh
javac --version 
javac 11.0.16.1
```